node/sc/local_backend.go

```diff
@@ -70,6 +70,14 @@ func (lb *LocalBackend) CodeAt(ctx context.Context, contract common.Address, blo
 	return statedb.GetCode(contract), nil
 }
 
+func (lb *LocalBackend) BalanceAt(ctx context.Context, account common.Address, blockNumber *big.Int) (*big.Int, error) {
+	if blockNumber != nil && blockNumber.Cmp(lb.subbrige.blockchain.CurrentBlock().Number()) != 0 {
```

Comment:
If blockNumber is nil, don't you handle it?

Reply:
```suggestion
	if blockNumber == nil || blockNumber.Cmp(lb.subbrige.blockchain.CurrentBlock().Number()) != 0 {
```

Reply:
If the blockNumber is nil, this will use current block. :) Thanks.